playground/module-resolution/vite.config.no-prebundling.ts

```diff
@@ -15,27 +15,17 @@ export default defineConfig({
 			external: ['@non-existing/pkg'],
 		},
 	},
-	plugins: [
-		cloudflare({
-			workers: {
-				worker: {
-					main: './src/index.ts',
-					wranglerConfig: './src/wrangler.toml',
-					// the following overrides partially opts out of prebundling
-					overrides: {
-						optimizeDeps: {
-							// we specifically opt-out of prebundling for the following dependencies
-							exclude: ['@cloudflare-dev-module-resolution/requires', 'react'],
-						},
-						resolve: {
-							// external modules don't get prebundled
-							external: ['@cloudflare-dev-module-resolution/requires/ext'],
-						},
-					},
-				},
+	environments: {
+		worker: {
+			optimizeDeps: {
+				// we specifically opt-out of prebundling for the following dependencies
+				exclude: ['@cloudflare-dev-module-resolution/requires', 'react'],
 			},
-			entryWorker: 'worker',
-			persistTo: false,
-		}),
-	],
+			resolve: {
+				// external modules don't get prebundled
+				external: ['@cloudflare-dev-module-resolution/requires/ext'],
+			},
+		},
+	},
+	plugins: [cloudflare({ viteEnvironmentName: 'worker' })],
```

Comment:
Not sure what you mean by 'inline the env configs'. Can you explain?

I do think it might be good for us to allow setting some Vite environment options in the plugin config but that seems like something different.

Reply:
with inline env config I mean setting the environment configs in the config option here, so instead of having:
```
environments: {
		worker: {
			<CONFIGS>
		},
},
plugins: [cloudflare({ viteEnvironmentName: 'worker' })], // <- the name is used to get the <CONFIGS>
```

it would be probably feel much more natural to me to have:
```
plugins: [cloudflare({ <CONFIGS> })], // (with an optional `viteEnvironmentName`)
```
or, to have a more clear distinction between configs have a specific field for this (probably better):
```
plugins: [cloudflare({ viteEnvironmentName: 'worker', environmentConfig: <CONFIGS> })],
```